File: benchmark/comparison/results/blind_ts/26.ts
function format_greeting(name: string, age: number): string {
  return `Hello, ${name}! You are ${age} years old.`;
}

function print_greeting(name: string, age: number): void {
  console.log("=== Greeting ===");
  console.log(format_greeting(name, age));
  console.log("================");
}

print_greeting("Alice", 30);
print_greeting("Bob", 25);

File: benchmark/comparison/results/blind_py/26.py
def format_greeting(name, age):
    return f"Hello, {name}! You are {age} years old."

def print_greeting(name, age):
    print("=== Greeting ===")
    print(format_greeting(name, age))
    print("================")

print_greeting("Alice", 30)
print_greeting("Bob", 25)
